src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -316,24 +320,32 @@ export default function PreferencesComponent() {
 					onChange={e => makeLocalChanges('defaultHelpUrl', e.target.value)}
 				/>
 			</div>
-
-			<Button
-				type='submit'
-				onClick={() =>
-					submitPreferences(localAdminPref)
-						.unwrap()
-						.then(() => {
-							showSuccessNotification(translate('updated.preferences'));
-						})
-						.catch(() => {
-							showErrorNotification(translate('failed.to.submit.changes'));
-						})
-				}
-				disabled={!hasChanges}
-				className='align-self-end mt-3'
-			>
-				{translate('submit')}
-			</Button>
+			<div className='d-flex justify-content-end mt-3'>
+				<Button
+					type='button'
+					onClick={discardChanges}
+					disabled={!hasChanges}
+					style={{ marginRight: '20px' }}
+				>
+					{translate('discard')}
```

Comment:
This is not changing languages. I think you want to use the key 'discard.changes' per data.ts and used in other files.